libs/wingc/src/jsify.rs

```diff
@@ -145,6 +145,33 @@ impl JSifier {
 			}
 		}
 	}
+	// note: Globals are emitted here and wrapped in "{{ ... }}" blocks. Wrapping makes these emissions, actual
+	// statements and not expressions. this makes the runtime panic if these are used in place of expressions.
+	fn jsify_global_utility_function(&self, args: &ArgList, function_type: UtilityFunctionKind) -> String {
+		match function_type {
+			UtilityFunctionKind::Assert => {
+				return format!(
+					"{{((cond) => {{if (!cond) throw new Error(`preflight assertion failed: '{0}'`)}})({0})}}",
```

Comment:
Why preflight? This should work for inflight as well, no?

Reply:
You are too quick! fb08498886bad95c2e00ca6ea8198d1c70cdade4